packages/twenty-server/src/engine/metadata-modules/field-metadata/services/field-metadata.service-v2.ts

```diff
@@ -56,6 +58,76 @@ export class FieldMetadataServiceV2 extends TypeOrmQueryService<FieldMetadataEnt
     return createdFieldMetadata;
   }
 
+  public async deleteOneField({
+    deleteOneFieldInput,
+    workspaceId,
+  }: {
+    deleteOneFieldInput: DeleteOneFieldInput;
+    workspaceId: string;
+  }): Promise<void> {
+    const { flatObjectMetadataMaps: existingFlatObjectMetadataMaps } =
+      await this.workspaceMetadataCacheService.getExistingOrRecomputeFlatObjectMetadataMaps(
+        {
+          workspaceId,
+        },
+      );
+
+    const flatFieldMetadatasToDelete =
+      fromDeleteFieldInputToFlatFieldMetadatasToDelete({
+        deleteOneFieldInput,
+        existingFlatObjectMetadataMaps,
+      });
+
+    const validationErrors = flatFieldMetadatasToDelete.flatMap(
+      (flatFieldMetadataToDelete) =>
+        this.flatFieldMetadataValidatorService.validateFlatFieldMetadataDeletion(
+          {
+            existingFlatObjectMetadataMaps,
+            flatFieldMetadataToDelete,
+          },
+        ),
+    );
+
+    if (validationErrors.length > 0) {
+      throw new MultipleMetadataValidationErrors(
+        validationErrors,
+        validationErrors.length > 1
+          ? 'Multiple validation errors occurred while deleting field'
```

Comment:
translate

Reply:
Will do a pass on all errors user friendlyness and translations at the end